applications/FluidDynamicsApplication/python_scripts/navier_stokes_compressible_solver.py

```diff
@@ -81,37 +82,37 @@ def __init__(self, main_model_part, custom_settings):
 
 
     def AddVariables(self):
-        
+
         self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.MOMENTUM)
         self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.DENSITY)
         self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.TOTAL_ENERGY)
-        
+
         self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.CONDUCTIVITY)
         self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.SPECIFIC_HEAT)
         self.main_model_part.AddNodalSolutionStepVariable(KratosFluid.HEAT_CAPACITY_RATIO)
-        
+
         self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.IS_STRUCTURE) ## ?
         self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.BODY_FORCE)
         self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.NODAL_H) ## ?
         self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.NODAL_AREA) ## ?
         self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.REACTION)  #for momentum
         self.main_model_part.AddNodalSolutionStepVariable(KratosFluid.REACTION_DENSITY)  #for momentum
```

Comment:
I'd take the chance to update the comments.

Reply:
To be honest, I think some of the marked variables may in fact be safely removed, but there is no test, so I didn't want to do it in this PR.